plugins/modules/packaging/os/apk.py

```diff
@@ -61,6 +61,11 @@
       - Upgrade all installed packages to their latest version.
     type: bool
     default: no
+  world:
+    description:
+     - Use a custom world file when checking for explicitly installed packages
+     type: str
+     default: /etc/apk/world
```

Comment:
```suggestion
     default: /etc/apk/world
     version_added: 5.4.0
```

Reply:
Great, will do. I noticed the indention errors just now, whoops.

Do you want me to amend and force-push?

Reply:
Adding new commits is also totally fine.